### PR TITLE
feat(database): setup db config

### PR DESCRIPTION
**Pull request by Lianathanoj**
### Context:
Continue on with parts of Miyagi.

### What:
* Setup database configuration files.
* Add migrations for movies
* Add seeding for movies table

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ node_js:
   - 6.11.1
 env:
   - NODE_ENV="test"
+before_script:
+  - yarn run db:setup:user
```
Comment (sjl2) [inline .travis.yml]: nit: I don't think you need to have `run` here or in the next line.

File: config/development.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 'use strict';
 
 module.exports = {
-  PORT: 3000
+  PORT: 3000,
+  DB_HOST: 'localhost',
```
Comment (sjl2) [inline config/development.js]: nit: alphabetize the entire exports object.

File: config/test.js [hunk 3]
```diff
@@ -1,5 +1,10 @@
 'use strict';
 
 module.exports = {
-  PORT: 3000
+  PORT: 3000,
```
Comment (sjl2) [inline config/test.js]: nit: alphabetize

**Review by sjl2 — APPROVED** [review]
LGTM with a couple nits

**Review by kienan — APPROVED** [review]
LGTM!

Try to practice writing good PR descriptions at this point though. With the 'Context` section, try to give the reviewer an idea of what's led up to this PR and maybe even what's coming next. You might also explain the scope of this PR, etc.

**Pull request merged**